Main.java

```diff
@@ -4,26 +4,42 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.StringTokenizer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class Main {
 
-    final static String FILE_TYPE_TXT = ".txt";
-    final static String FILE_TYPE_JSON = ".json";
-    final static String REMOVE_SPECIAL_CHAR = "\\p{Punct}";
-    final static String SINGLE_SPACE = "";
+    static final String FILE_TYPE_TXT = ".txt";
+    static final String FILE_TYPE_JSON = ".json";
+    static final String REMOVE_SPECIAL_CHAR = "\\p{Punct}";
+    static final String SINGLE_SPACE = "";
+    static final String EXCLUDE_SPECIAL_CHAR = "[a-zA-Z0-9\"\"]";
 
     public static void main(String[] args) {
 
-        if (args.length != 2) {
+        if (args.length < 2) {

```

Comment:
I believe you don't need to check less than and greater than 2 args separately, rather check for !equal to 2 and print message like Program accepts only two arguments-filepath and search word.. something like this.

Reply:
Sure. I will do that.